console/war/src/main/java/org/frankframework/runner/ConsoleStandaloneInitializer.java

```diff
@@ -28,13 +35,23 @@
  */
 public class ConsoleStandaloneInitializer {
 
+	public static class WsSciWrapper implements ServletContextInitializer {
```

Comment:
Required to enable the use of websockets when starting as (spring)bootable application.